tests/unit/test_fracture_network.py

```diff
@@ -1,16 +1,115 @@
 """
-Various checks of the FractureNetwork2d and 3d class
+Various checks for FractureNetwork2d, FractureNetwork3d, and create_fracture_newtork().
 
-Also test unitily function for generation of defalut domains.
+Also tests for utility functions for generation of default domains.
 """
 from __future__ import annotations
 
 import unittest
+import pytest
 
 import numpy as np
 
 import porepy as pp
 from tests import test_utils
+from porepy.fracs.utils import pts_edges_to_linefractures
+from porepy.fracs.fracture_network_2d import FractureNetwork2d
+from porepy.fracs.fracture_network_3d import FractureNetwork3d
+from porepy.grids.standard_grids.utils import unit_domain
+
+
+class TestCreateFractureNetwork:
```

Comment:
If we raise an error if domain and fracture dimension differ, we would need to add a test for this.

Reply:
Added.